src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -34,6 +34,9 @@ public class TextPresenter : Control
         public static readonly StyledProperty<IBrush?> CaretBrushProperty =
             AvaloniaProperty.Register<TextPresenter, IBrush?>(nameof(CaretBrush));
 
+        public static readonly StyledProperty<TimeSpan> CaretBlinkIntervalProperty =
+            AvaloniaProperty.Register<TextPresenter, TimeSpan>(nameof(CaretBlinkInterval), defaultValue: TimeSpan.FromMilliseconds(500));
```

Comment:
You can use AddOwner here so the property is only defined at one location

Reply:
Fixed. Uses AddOwner now.